docs/gitalb/gitlab-gcp.md

```diff
@@ -0,0 +1,380 @@
+# Deploying GitLab in GCP
```

Comment:
This file is in directory `gitalb`, not `gitlab`.

Reply:
:heavy_check_mark: